packages/core/src/rules/prefer_is_not.ts

```diff
@@ -50,13 +53,48 @@ IF NOT variable = 42.`,
           continue;
         }
 
-        const message = "Prefer IS NOT to NOT IS";
-        issues.push(Issue.atToken(file, c.getFirstToken(), message, this.getMetadata().key, this.conf.severity));
+        const message = "Prefer expresion NOT to NOT expresion";
+
+        const fix = this.getFix(file, c);
+        if (fix === undefined) {
+          issues.push(Issue.atToken(file, c.getFirstToken(), message, this.getMetadata().key, this.conf.severity));
+        }
+        else {
+          issues.push(Issue.atToken(file, c.getFirstToken(), message, this.getMetadata().key, this.conf.severity, fix));
+        }
```

Comment:
think this will work,
 
```suggestion
        issues.push(Issue.atToken(file, c.getFirstToken(), message, this.getMetadata().key, this.conf.severity, fix));
```